homework_1/decorators/cache_decorator.py

```diff
@@ -1,9 +1,9 @@
 import functools
-import requests
 
 
 def lfu_cache(max_limit=64):
-    """Decorator for caching with deleting elements by LFU algorithm after going beyond the limit"""
+    """A decorator for caching with deleting elements by LFU algorithm after going beyond the limit"""
+
     def internal(func):
         @functools.wraps(func)
```

Comment:
remove comments, it's weird to comment on each line.
And also dont use russian lanuage comments, only English.

Reply:
✅